README.md

```diff
@@ -84,4 +93,19 @@ return {
 };
 ```
 
-See [the Bugsnag docs](https://bugsnag.com/docs/notifiers/js#metadata) for more information.
+### Identifying Users ([docs](https://docs.bugsnag.com/platforms/browsers/#identifying-users))
+
+To correlate a specific user to an error and have the information appear in the
+User tab in the Bugsnag UI, send user data with each error data. Define a
+helper method `getUser` in the `app/utils/bugsnag.js` you created. `getUser`
+takes the container as an argument, e.g.:
+
+```js
+export function getUser(container) {
+  return {
+    id: 7,
+    name: 'Conrad Irwin',
+    email: 'conrad@bugsnag.com',
```

Comment:
Great docs, thanks for adding them and corrections.

Minor style point: Trailing comma here.